lib/index.js

```diff
@@ -30,10 +31,17 @@ Outbound.prototype.initialize = function() {
   window.outbound.methods = [
     'identify',
     'track',
+    'alias',
     'registerApnsToken',
     'registerGcmToken',
     'disableApnsToken',
     'disableGcmToken',
+    'disableAllGcmTokens',
```

Comment:
@Tang8330 sorry, but to clarify it sounds like you test locally against ie9 and it worked?